src/tools/loggers/consoleLogger.js

```diff
@@ -0,0 +1,11 @@
+import { logLevels } from '@tvkitchen/base-constants'
+
+export default {
```

Comment:
Some thoughts:

1. This feels a little like a DRY violation, but maybe not one that would be better if it weren't.
2. Do we want to output errors to the error output stream?
3. We're depending on the context this is called in to define the logger in a property called "logger." Are we sure we want to couple it that way?
4. Most of this is defined by the built-in logger, except `fatal` and prepending the level to the message. Theoretically we could just pass in `console` except for those requirements.

Reply:
1. which part?
2. Oh good thought (also, good thought re #4...)
3. Nope that is totally a mistake!  Also I should have tests which would have exposed that mistake.
4. Yes!  I'll go do that.

Reply:
OK I grok the DRY comment now.

Worth noting that if we just took `FATAL` out of `logLevels` we would be able to just use `console` as a default logger without needing a logger class anywhere...

The difference between `FATAL` and `ERROR` is the scope of the error.  If it takes down the whole service, it causes FATAL; if it's just taking down the local operation, it's ERROR...

Not sure we'd be giving up much by dropping fatal tbh.

Reply:
Redid this with a little inspiration from how [kafkajs](https://github.com/tulios/kafkajs/blob/master/src/loggers/console.js) does it -- I think I avoided a bit of the DRY smells but I imagine it isn't perfect.

Overall I am comfortable with the idea that we will probably want to revisit this -- both to add better structure, probably to remove `fatal` and the like as well depending on how the [conversation in base-constants goes](https://github.com/tvkitchen/base/issues/84).